Octokit/Clients/OrganizationMembersClient.cs

```diff
@@ -37,6 +37,20 @@ public enum OrganizationMembersRole
         Member
     }
 
+    public enum OrganizationMembershipRole
```

Comment:
Normally we declare enums in the Response/Request model area rather than directly in the client dont we?